pychromecast/discovery.py

```diff
@@ -166,6 +171,47 @@ def callback(uuid, name):
     return (listener.devices, browser)
 
 
+def discover_listed_chromecasts(
```

Comment:
One more discovery method 😆 Should we remove some other ones?

Reply:
I think it's fairly OK:
 - `discover_chromecasts` and `discover_listed_chromecasts` are helpers for discovery only.
 - `get_chromecasts` and `get_listed_chromecasts` are discovers and creates Chromecast objects.